jobs/kibana/spec

```diff
@@ -11,9 +10,9 @@ templates:
   helpers/ctl_setup.sh: "helpers/ctl_setup.sh"
   helpers/ctl_utils.sh: "helpers/ctl_utils.sh"
 properties:
-  kibana.elasticsearch:
-    description: "IP address and port of elasticsearch host to proxy requests for (eg, 127.0.0.1:9200)"
-    default: "http://127.0.0.1:9200"
+  kibana.api_host:
```

Comment:
Can we keep this as elasticsearch? Since kibana doesn't need to know the difference on whether it's logsearch's api or elasticsearch's api, it seems better to stick with expected vendor conventions.